src/main/scala/org/dbpedia/spotlight/wikistats/ComputeStats.scala

```diff
@@ -103,7 +101,7 @@ class ComputeStats(lang: String) (implicit val sc: SparkContext,implicit val sql
     val totalSfDf = totalSfsRDD.toDF("wid", "sf2")
     val uriSfDf = wikipediaParser.getSfURI().toDF("wid", "sf1", "uri")
 
-    (totalSfDf.persist(StorageLevel.MEMORY_ONLY_SER),uriSfDf.persist(StorageLevel.MEMORY_ONLY_SER))
+    (totalSfDf.persist(StorageLevel.MEMORY_AND_DISK),uriSfDf.persist(StorageLevel.MEMORY_AND_DISK))
```

Comment:
I wonder if this whole "lets do jsonwiki + stats" at the same time its making things harder for us.
As instead of generating a jsonwiki first and working the stats from there